auth/src/main/java/com/firebase/ui/auth/util/ui/PreambleHandler.java

```diff
@@ -41,7 +41,14 @@ public PreambleHandler(Context context, FlowParameters parameters, @StringRes in
         setupCreateAccountPreamble();
```

Comment:
Should we move this call out of the constructor and into the static `setup` method as well?

Reply:
Good idea, done.